plugin/src/test/java/io/jenkins/plugins/casc/core/JdkConfiguratorTest.java

```diff
@@ -35,8 +30,12 @@ public void should_configure_maven_tools_and_global_config() {
         InstallSourceProperty installSourceProperty = jdk.getProperties().get(InstallSourceProperty.class);
         Assert.assertEquals(1, installSourceProperty.installers.size());
 
+        // https://github.com/jenkinsci/jenkins/pull/3301 moved jdk-tool
```

Comment:
remove this code?

Reply:
Done, wasn't sure of the policy, just wanted to call out why I had to do this.